### ISSUE TITLE
Change default for resource parameters

### ISSUE THREAD

**Issue by gustavdelius** [sizespectrum/mizer#185]
When setting up a new model, I propose that the basic resource replenishment rate should be chosen so that at steady state the plankton is at about half of its carrying capacity. This means that the replenishment rate should be similar to the per-capita resource mortality rate. This of course depends on the abundance of fish. The current default is chosen independently of the abundance of fish and therefore in some models can be very wrong.

**Comment by Kenhasteandersen** [comment]
I agree that we need to look into the replenishment rate such that there is a depletion of the resource.

 This means that the replenishment rate should be similar to the per-capita resource mortality rate. This of course depends on the abundance of fish. The current default is chosen independently of the abundance of fish and therefore in some models can be very wrong.

I’m not sure this is entirely correct. The mortality depends on the ratio between the resource level and the fish abundance. This ratio is roughly independent of fish abundance (as abundance scales with the resource level). Therefore the replenishment rates should be roughly independent of fish abundance. Right?

Ken

**Comment by gustavdelius** [comment]
Hi @Kenhasteandersen . Why do you say that the mortality depends on the ratio between the resource level and the fish abundance? Because this issue tracker is not very good for maths equations, I have written down what I mean at https://www.overleaf.com/read/wjhmczpmcmmz

**Comment by Kenhasteandersen** [comment]
From the equation, it is because gamma is set to be proportional to 1/kappa to achieve the desired feeding level.

There is also an intuitive argument: Let’s say we have one prey and one predator, and the predator has a 50% chance of eating the prey in a day. The mortality is then roughly 0.5/day. If there are 10 prey, the mortality declines to 0.05%; the risk of being eaten as a prey is smaller if you are many.

Ken

---------------------------------------------------------------------------------------------------------------
Ken H. Andersen, http://ken.haste.dk<http://ken.haste.dk/>, twitter: @69kno
Professor in theoretical marine ecology, head of section and phd school, and deputy director of Centre for Ocean Life http://www.oceanlifecentre.dk<http://www.oceanlifecentre.dk/>

Try my plankton simulator: http://oceanlife.dtuaqua.dk/Plankton/

New book: Fish Ecology, Evolution, and Exploitation - A New Theoretical Synthesis https://press.princeton.edu/titles/13516.html



***@***.***D09493.A377DFB0]

On 23 Mar 2021, at 22.16, Gustav W Delius ***@***.******@***.***>> wrote:



Hi @Kenhasteandersen<https://github.com/Kenhasteandersen> . Why do you say that the mortality depends on the ratio between the resource level and the fish abundance? Because this issue tracker is not very good for maths equations, I have written down what I mean at https://www.overleaf.com/read/wjhmczpmcmmz

**Comment by gustavdelius** [comment]
This is now obsolete by #245
